frontend/src/components/templates/Stats.js

```diff
@@ -108,15 +108,22 @@ export const Stats = () => {
               <Tag type={isActive ? 'info' : ''}>Toutes les APIs</Tag>
             )}
           </NavLink>
-          {DATA_PROVIDER_WITH_ENROLLMENTS_IN_PRODUCTION_ENV.map((targetApi) => (
-            <NavLink key={targetApi} end to={`/stats/${targetApi}`}>
-              {({ isActive }) => (
-                <Tag type={isActive ? 'info' : ''}>
-                  {DATA_PROVIDER_PARAMETERS[targetApi]?.label}
-                </Tag>
-              )}
-            </NavLink>
-          ))}
+          {Object.keys(DATA_PROVIDER_PARAMETERS)
```

Comment:
can you put this in a separate variable ? it's quite a load of logic to be put within jsx